github/Repository.py

```diff
@@ -2349,18 +2373,27 @@ def delete_file(
         committer=github.GithubObject.NotSet,
         author=github.GithubObject.NotSet,
     ):
-        """This method deletes a file in a repository
+        """This method deletes a file in a repository.
 
-        :calls: `DELETE /repos/{owner}/{repo}/contents/{path} <https://docs.github.com/en/rest/reference/repos#delete-a-file>`_
+        :calls:`DELETE /repos/{owner}/{repo}/contents/{path}
+            <https://docs.github.com/en/rest/reference/repos#delete-a-
+            file>`_
         :param path: string, Required. The content path.
         :param message: string, Required. The commit message.
-        :param sha: string, Required. The blob SHA of the file being replaced.
-        :param branch: string. The branch name. Default: the repository’s default branch (usually master)
-        :param committer: InputGitAuthor, (optional), if no information is given the authenticated user's information will be used. You must specify both a name and email.
-        :param author: InputGitAuthor, (optional), if omitted this will be filled in with committer information. If passed, you must specify both a name and email.
-        :rtype: {
-            'content': :class:`null <github.GithubObject.NotSet>`:,
-            'commit': :class:`Commit <github.Commit.Commit>`}
+        :param sha: string, Required. The blob SHA of the file being
+            replaced.
+        :param branch: string. The branch name. Default: the
+            repository’s default branch (usually master)
+        :param committer: InputGitAuthor, (optional), if no information
+            is given the authenticated user's information will be used.
+            You must specify both a name and email.
+        :param author: InputGitAuthor, (optional), if omitted this will
+            be filled in with committer information. If passed, you must
+            specify both a name and email.
+        :rtype: { 'content'
+        : : class:`null <github.GithubObject.NotSet>`:, 'commit'
+        : : class:`Commit <github.Commit.Commit>`}
```

Comment:
ok, this also seems wrong

Reply:
ok, I think in this case the docstring shall be marked with r to be parsed as RST